app/managers.py

```diff
@@ -0,0 +1,45 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("../cinema.sqlite")
```

Comment:
Consider parameterizing the database connection string or configuring it externally for better flexibility and security. Hardcoding paths can lead to issues when deploying or testing in different environments.